android/app/src/main/java/app/organicmaps/util/SharingUtils.java

```diff
@@ -127,6 +126,13 @@ public static ActivityResultLauncher<Intent> RegisterLauncher(@NonNull Fragment
   }
   public static void shareBookmarkFile(Context context, ActivityResultLauncher<Intent> launcher, String fileName)
   {
+    final String ext = fileName.substring(fileName.lastIndexOf("."));
+    final String mimeType = switch (ext) {
```

Comment:
@rtsisyk a .kml extension may also be introduced later in the future, it is easy to handle this case now to avoid surprises later.